sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateClientTestBase.java

```diff
@@ -283,6 +280,19 @@ void listCertificatesRunner(Consumer<List<String>> testRunner) {
         testRunner.accept(certificates);
     }
 
+    @Test
+    public abstract void listPropertyOfCertificates(HttpClient httpClient, CertificateServiceVersion serviceVersion);
+
+    void listPropertyOfCertificatesRunner(Consumer<List<String>> testRunner){
```

Comment:
I don't know if we need different runners for overloads of the same method if they do the same thing. I guess it's ok to leave it as is but may be a good thing to consider in a future revision.